packages/components/src/avatar/src/AvatarGroup.tsx

```diff
@@ -9,6 +9,7 @@ import { Tooltip, TooltipTrigger } from "../../tooltip";
 const DefaultElement = "div";
 
 export interface InnerAvatarGroupProps extends Omit<AbstractGroupProps<typeof DefaultElement>, "gap" | "orientation"> {
+    remainingAvatarsProps?: any;
```

Comment:
Note for myself: 
Is this prop ok? was there a comment about it earlier?

The typing is wrong here